src/main/java/org/jitsi/videobridge/VideoChannel.java

```diff
@@ -494,8 +494,6 @@ void sctpConnectionReady(Endpoint endpoint)
                 sendLastNEndpointsChangeEventOnDataChannel(
                         lastNController.getForwardedEndpoints(), null);
             }
-
-            updateInLastN(this);
```

Comment:
This one I don't have the best understanding of.. but I was seeing an issue when lastN=1 and clients A and B are in a conference. Then a 3rd client, C, tries to join.

Client C downloads 1 stream like we expect and can switch between A and B's feeds, but A or B cannot view C's stream (even when pinned)

It looks like this call sets C's inLastN value to false for all the other channels, which causes the LastNTransformEngine to drop all the RTP packets. 

This results in A and B receiving a muted video stream from client C. 

Is there somewhere that the inLastN value should be set back to true?